magic/oracle.py

```diff
@@ -231,3 +231,24 @@ def insert_scryfall_card(sfcard):
         multiverse.insert_card(c)
     multiverse.update_cache()
     CARDS_BY_NAME[sfcard['name']] = load_card(sfcard['name'])
+
+def last_pd_rotation_changes():
+    current_code = rotation.last_rotation_ex()['code']
+    previous = multiverse.SEASONS[multiverse.SEASONS.index(current_code) - 1]
+    previous_id = multiverse.get_format_id("Penny Dreadful {f}".format(f=previous))
+    current_id = multiverse.get_format_id("Penny Dreadful")
+    return changes_between_formats(previous_id, current_id)
+
+def changes_between_formats(f1, f2):
+    return [query_diff_formats(f2, f1), query_diff_formats(f1, f2)]
+
+def query_diff_formats(f1, f2):
+    query = 'c.id IN (SELECT card_id FROM card_legality WHERE format_id = {format1}) AND c.id NOT IN (SELECT card_id FROM card_legality WHERE format_id = {format2})'
```

Comment:
Nit. I like to use triple-quoted strings to layout SQL is a more readable way. Not important.